grammar/haxe.bnf

```diff
@@ -527,20 +537,18 @@ left shiftExpression ::= shiftOperator additiveExpressionWrapper
 private additiveExpressionWrapper ::= multiplicativeExpressionWrapper additiveExpression*
 left additiveExpression ::= ('+' | '-') multiplicativeExpressionWrapper
 
-private multiplicativeExpressionWrapper ::= assignExpressionWrapper multiplicativeExpression*
-left multiplicativeExpression ::= ('*' | '/' | '%') assignExpressionWrapper
+private multiplicativeExpressionWrapper ::= prefixExpression multiplicativeExpression*
+left multiplicativeExpression ::= ('*' | '/' | '%') (prefixExpression | suffixExpressionWrapper)
 
 /* Technically, assignExpressions don't have higher priority than everything else.
```

Comment:
Hmm.  Or maybe it _should_ stay, given that you've added `assignExpression` to `suffixExpressionWrapper` at line 543.  Given that you had to do that, it seems that the precedence is still incorrect.  What am I missing?